pkg/ottl/ottlfuncs/README.md

```diff
@@ -1519,6 +1519,16 @@ Examples:
 - `Time("2012-11-01T22:08:41+0000 EST", "%Y-%m-%dT%H:%M:%S%z %Z")`
 - `Time("2023-05-26 12:34:56", "%Y-%m-%d %H:%M:%S", "America/New_York")`
 
+`locale` specifies the input language of the `target` value. It is used to interpret timestamp values written in a specific language, 
```

Comment:
Please add something that conveys that when not set english is the local that gets used.